prow/cmd/peribolos/main_test.go

```diff
@@ -2773,6 +2773,84 @@ func TestConfigureRepos(t *testing.T) {
 			repos:         []github.Repo{{Name: oldName, Private: true}},
 			expectedRepos: []github.Repo{{Name: oldName, Private: false}},
 		},
+		{
+			description: "renaming a repo is successful",
+			orgConfig: org.Config{
+				Repos: map[string]org.Repo{
+					newName: {Previously: []string{oldName}},
+				},
+			},
+			repos:         []github.Repo{{Name: oldName, Description: "renamed repo"}},
+			expectedRepos: []github.Repo{{Name: newName, Description: "renamed repo"}},
+		},
+		{
+			description: "renaming a repo by just changing case is successful",
+			orgConfig: org.Config{
+				Repos: map[string]org.Repo{
+					"repo": {Previously: []string{"REPO"}},
```

Comment:
I actually think that if I declare `Repo` and the current name is `rEpO` then we should rename it to `Repo` -- without requiring a previously.

Reply:
Makes sense, will fix!

Reply:
It actually worked like this in the PR. I just added a test for it.